cerberus/errors.py

```diff
@@ -459,9 +460,15 @@ def clear(self):
         self.tree = {}
 
     def format_message(self, field, error):
-        return self.messages[error.code]\
-            .format(*error.info, constraint=error.constraint,
-                    field=field, value=error.value)
+        message = self.messages[error.code]
+        try:
+            if isinstance(error.value, unicode):
+                message = unicode(message)
+        except NameError:
```

Comment:
Agree, since it's code specific to Python version implicit check for current Python version would be better. Unless this part targeting the version could be moved outside to `platform` module.